packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Interaction Code grant
+meta:
+  - name: description
+    content: An overview of the Interaction Code grant type for Okta Identity Engine.
+---
+
+# Interaction Code grant
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+## Overview
+
+To enable a more customized user authentication experience, Okta has introduced an addition to the [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid) standard called the Interaction Code grant type. This grant type allows apps to manage user interaction with the Okta Authorization Server without a browser. This is useful when the client has a particular way that it wants to interact with the user and doesn’t need to share an authenticated session with other applications.
+
+The Interaction Code flow consists of a series of interactions between the user and the Authorization Server, facilitated by the client. Each interaction is called a remediation step and corresponds to a piece of user data required by the Authorization Server. The client obtains these remediation steps from the Identity Engine component of the Authorization Server and prompts the user for the required data to continue the flow.
+
+For example, a user could start an authentication flow by entering only a username, and this flow would prompt the client to request more information, or remediation, as required. Remediation is the direct communication between the client and the Authorization Server without a browser redirect. An example of a remediation step is the client prompting the user for a password or to add a second factor and then sending that information directly to the Authorization Server.
```

Comment:
`Remediation is the direct communication between the client and the Authorization Server **without a browser redirect.**`

The remediation could be to transition to a pushed authorization request, which would require a redirect